examples/tls/tls_server.c

```diff
@@ -374,11 +385,19 @@ int TPM2_TLS_ServerArgs(void* userCtx, int argc, char *argv[])
             goto exit;
         }
 
+        rc = wc_EccPublicKeyToDer(&wolfEccKey, der, derSz, 1);
+        if (rc < 0) {
+            printf("Failed to export ECC public key!\n");
+            goto exit;
+        }
+        derSz = rc;
+        rc = 0;
+
         /* Private key is on TPM and crypto dev callbacks are used */
-        /* TLS server requires some dummy key loaded (workaround) */
-        if (wolfSSL_CTX_use_PrivateKey_buffer(ctx, DUMMY_ECC_KEY,
-                sizeof(DUMMY_ECC_KEY), WOLFSSL_FILETYPE_ASN1) != WOLFSSL_SUCCESS) {
-            printf("Failed to set key!\r\n");
+        /* TLS client (mutual auth) requires a public key loaded */
```

Comment:
The API is for using a private key, so this comment "LOOKS" like a mistake.  Can we add something to say that we're doing this on purpose? Something like "Even though this is API is for a private key, ....."